lib/solid_queue.rb

```diff
@@ -6,6 +6,9 @@
 require "active_job/queue_adapters/solid_queue_adapter"
 require "active_job/concurrency_controls"
 
+require "active_support/deprecator"
```

Comment:
Why do you need this one?

Reply:
I hit an error in an IRB console trying to only `require "active_support/core_ext/numeric/time"`. However, I think I'll just close this for now, as this is really separate from my primary goal: to create a bug report script guide. I think I just need to figure out what all `require`s are necessary at the top of that script. And changes to the core lib are very likely not needed; just the right collection of `require`s at the top of that script

Reply:
Thank you @fractaledmind! I'm planning to switch to [zeitwerk](https://github.com/fxn/zeitwerk) soon, I just haven't had the time yet, so hopefully you won't run into this again.